lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -24,10 +26,12 @@ class DSBottomSheetService {
       child: Column(
         mainAxisSize: MainAxisSize.min,
         children: [
-          Visibility(
```

Comment:
you can use the replacement property of visibility widget, to show container, when hide grabber is true